app/src/main/java/com/pmatuki/wowpapers/WowpaperApplication.kt

```diff
@@ -0,0 +1,26 @@
+package com.pmatuki.wowpapers
+
+import android.app.Application
+import android.content.Context
+import toothpick.config.Module
+import toothpick.ktp.KTP
+import toothpick.ktp.binding.bind
+
+class WowpaperApplication : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+        KTP.openRootScope().openSubScope(APP_SCOPE).installModules(AppModule(this)).inject(this)
+    }
+
+    class AppModule(application: Application) : Module() {
+        init {
+            val context = application.applicationContext
+            bind<Context>().toInstance(context)
```

Comment:
Yes, that's the idea. Based on the application we register an `Context` to be used across the application. 😊